inc/send-education.php

```diff
@@ -30,7 +30,9 @@ function send_edu() {
     $body .= "<br><br>" . esc_html($input);
 
     if($email !== '') {
-			$sender = "$name <$email>";
+      if(!isset($_POST['anonymous'])) {
```

Comment:
Hmmmm volgens mij kan dit wel beter in dezelfde `if`, anders gebeurt er nu niks (is er geen sender) als `$email` leeg is maar `$_POST['anonymous']` `true` is. Dus dan zou het worden:

```
    if($email !== '' && !isset($_POST['anonymous'])) {
      $sender = "$name <$email>";
    } else {
      $sender = "Anonymous <anonymous@svid.nl>";
    }
```